index.d.ts

```diff
@@ -8,6 +8,11 @@ declare namespace getPort {
 		*/
 		readonly port?: number | Iterable<number>;
 
+		/**
+		A port or list of ports which we do not want to use
+		*/
+		readonly exclusions?: number | Array<number>;
```

Comment:
It should not support a single port.

Reply:
Curious on this, why remove the flexibility?

Reply:
@sindresorhus  so I understand you correctly, we want `exclude` to only support an array of ports to exclude , not just a single port value.

Reply:
Almost. It should accept an `Iterable`.

Reply:
Aaa I see now, changed it to be an Iterable!